pylint/checkers/base_checker.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, linter=None):
 
     def __gt__(self, other):
         """Permit to sort a list of Checker by name."""
-        return f"{self.name}{self.msgs}".__gt__(f"{other.name}{other.msgs}")
+        return f"{self.name}{self.msgs}" > (f"{other.name}{other.msgs}")
```

Comment:
This is uncovered. We might want to do a follow up issue/PR and see if this is necessary anyway.

Reply:
Noted this in #6045 👍

Reply:
Thanks! And thank you for being so proactive these last couple of days! Real breeze of fresh air in the project 😄